vm.c

```diff
@@ -429,7 +429,7 @@ jit_compile(rb_execution_context_t *ec)
     if (body->jit_entry == NULL) {
         body->jit_entry_calls++;
         if (yjit_enabled) {
-            if (body->jit_entry_calls == rb_yjit_call_threshold())  {
+            if (rb_yjit_threshold_hit(iseq, body->jit_entry_calls)) {
```

Comment:
I think you want to make this change to `jit_compile_exception` as well.

Reply:
pushed https://github.com/Shopify/ruby/commit/1c0d2a2075b64b1930bd07f7b5b3a441284fd775

Reply:
Reverting for now.